src/domain/challenge/challenge/challenge.resolver.mutations.ts

```diff
@@ -45,10 +45,10 @@ export class ChallengeResolverMutations {
   @Profiling.api
   async createChildChallenge(
     @CurrentUser() agentInfo: AgentInfo,
-    @Args('challengeData') challengeData: CreateChallengeInput
+    @Args('challengeData') challengeData: CreateChallengeInChallengeInput
```

Comment:
Won't it be better to name it CreateSubChallengeInput or CreateChildChallengeInput?